content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -250,19 +295,30 @@ can run at the same time.
 This is a significant undertaking, and almost all Kubernetes users find they
 do not need to modify the scheduler.
 
-The scheduler also supports a
+You can control which [scheduling plugins](/docs/reference/scheduling/config/#scheduling-plugins)
+are active, or associate sets of plugins with different named [scheduler profiles](/docs/reference/scheduling/config/#multiple-profiles).
+You can also write your own plugin that integrates with one or more of the kube-scheduler's
+[extension points](/docs/concepts/scheduling-eviction/scheduling-framework/#extension-points).
+
+Finally, built in `kube-scheduler` component also supports a
```

Comment:
nit: Finally, the built-in `kube-scheduler` component supports a [webhook]() that ...